src/pybind/mgr/orchestrator.py

```diff
@@ -1305,9 +1305,8 @@ class ServiceSpec(object):
 
     """
 
-    def __init__(self, service_type, name=None, placement=None):
```

Comment:
Sorry for the misunderstanding here! Having a `service_type` within `ServiceSpec` does make sense. As it makes it possible to just do `ceph orch import -i my_cluster.yaml` at some point, which is awesome! But it loosk like as if we need to rethink the class structure